### PR TITLE
Bump v0.25.0-1

### PR DESCRIPTION
**Pull request by arg0d**

### CODE REVIEW

File: compatibility-test/tests/go/coverall_test.go [hunk 1]
```diff
@@ -2,187 +2,193 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
```
Comment (arg0d) [inline compatibility-test/tests/go/coverall_test.go:2]: I'm not sure what happened here. At first I though it was tabs/spaces, but even when matching tabs/spaces, it was still showing diff. I guess it was newlines? :thinking:

File: compatibility-test/tests/cs/test_coverall.cs [hunk 2]
```diff
@@ -70,7 +72,7 @@ public class TestCoverall {
             Assert.Throws<PanicException>(() => coveralls.FalliblePanic("Expected panic in a fallible function!"));
 
             coveralls.TakeOther(null);
-            Assert.Equal(2UL, coveralls.StrongCount());
+            Assert.Equal(2UL, coveralls.StrongCount());
```
Comment (Lipt0nas) [inline compatibility-test/tests/cs/test_coverall.cs:75]:
```suggestion
            Assert.Equal(2UL, coveralls.StrongCount());
```

**Review by Lipt0nas — APPROVED** [review]

**Review by Savolro — CHANGES_REQUESTED** [review]

**Review by Savolro — APPROVED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
-RUN cargo install uniffi-bindgen-go --tag v0.1.5+v0.23.0 --git https://github.com/NordSecurity/uniffi-bindgen-go
 
-FROM debian:bullseye
+# Download crates.io index, saves time if subsequent steps change
+RUN cargo search --limit 0
+
+RUN cd /project && cargo install --path uniffi-bindgen
+RUN cargo install uniffi-bindgen-cs --tag v0.7.0+v0.25.0 --git https://github.com/NordSecurity/uniffi-bindgen-cs
+RUN cargo install uniffi-bindgen-go --tag v0.2.0+v0.25.0 --git https://github.com/NordSecurity/uniffi-bindgen-go
+RUN cargo install uniffi-bindgen-cpp --tag v0.1.0+v0.25.0 --git https://github.com/NordSecurity/uniffi-bindgen-cpp
```
[the remark this answers is not in the document]
Comment (arg0d) [reply Dockerfile:10]: This ballooned from my initial basic change to just add C++ :sweat_smile: Gonna keep it like this, and as we discussed in PM investigate the option to deduplicate `compatibility-test` to use upstream code directly without copying the tests into this repo.

[the remark this answers is not in the document]
Comment (arg0d) [reply Dockerfile:10]: https://github.com/NordSecurity/uniffi-generators/issues/10

**Pull request merged**